docs/docs/compliance/compliance.md

```diff
@@ -26,9 +26,11 @@ Once the report has been generated, you can fetch and review its results section
 kubectl get compliance nsa  -o=jsonpath='{.status}' | jq .
 ```
 
-## You can create your own custom compliance report. A compliance report is a simple YAML document in the following format:
+## Custom compliance reports
 
-<summary>customer compliance report</summary>
+You can create your own custom compliance report. A compliance report is a simple YAML document in the following format:
+
+<summary>Customer compliance report</summary>
```

Comment:
I think we can remove this : `<summary>Customer compliance report</summary>` as well

Reply:
Done.